data/ledger.go

```diff
@@ -118,25 +118,21 @@ func LoadLedger[T string | ledger.DirsAndPrefix](
 	return l, nil
 }
 
-// AddressTxns returns the list of transactions to/from a given address in specific round
-func (l *Ledger) AddressTxns(id basics.Address, r basics.Round) ([]transactions.SignedTxnWithAD, error) {
+// TxnsFrom returns the list of transactions to/from a given address in specific round
```

Comment:
Doesn't this comment imply also checking the receivers of transactions?

Reply:
The comment implied that, but the only callers are two stateproof endpoints, that are just trying to find stateproof transactions by looking for the stateproof sender.

Reply:
Oh, I see, yes, I should change the remaining comment.